Mathlib/Topology/MetricSpace/Dilation.lean

```diff
@@ -264,6 +265,20 @@ variable [DilationClass F α β] [DilationClass G β γ]
 
 variable (f : F) (g : G) {x y z : α} {s : Set α}
 
+/-- Every isometry is a dilation of ratio `1`. -/
+@[simps]
+def _root_.Isometry.toDilation {f : α → β} (hf : Isometry f) : α →ᵈ β where
```

Comment:
It might make sense to make `f` explicit, otherwise this shows up as `_.toDilation` in the goal view.